request-handler/src/main/java/au/org/aodn/aws/wps/lambda/WpsLambdaRequestHandler.java

```diff
@@ -0,0 +1,146 @@
+package au.org.aodn.aws.wps.lambda;
+
+
+import au.org.aodn.aws.wps.AwsApiRequest;
+import au.org.aodn.aws.wps.AwsApiResponse;
+
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.kms.AWSKMS;
+import com.amazonaws.services.kms.AWSKMSClientBuilder;
+import com.amazonaws.services.kms.model.DecryptRequest;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectInputStream;
+import com.amazonaws.util.Base64;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.Properties;
+
+public class WpsLambdaRequestHandler implements RequestHandler<AwsApiRequest, AwsApiResponse> {
+
+    private LambdaLogger LOGGER;
+    private static final String DEFAULT_ENV_NAME = "$LATEST";
+    private static final String ACCESS_KEY_ENV_VARIABLE_NAME = "ACCESS_KEY";
+    private static final String SECRET_KEY_ENV_VARIABLE_NAME = "SECRET_KEY";
+    private static final String REGION_NAME_ENV_VARIABLE_NAME = "S3_AWS_REGION";
+    private static final String S3_BUCKET_ENV_VARIABLE_NAME = "CONFIG_S3_BUCKET";
+    private static final String CONFIG_FILENAME_ENV_VARIABLE_NAME = "CONFIG_FILENAME";
+
+    @Override
+    public AwsApiResponse handleRequest(AwsApiRequest request, Context context) {
+        LOGGER = context.getLogger();
+
+        au.org.aodn.aws.wps.RequestHandler handler = new au.org.aodn.aws.wps.RequestHandler();
+        AwsApiResponse response = null;
+
+        //  Read environment variables
+        //  Identify the location of the configuration file to use
+        String accessKey = getEnvironmentVariable(ACCESS_KEY_ENV_VARIABLE_NAME);
+        String secretKey = getEnvironmentVariable(SECRET_KEY_ENV_VARIABLE_NAME);
+        String s3RegionName = getEnvironmentVariable(REGION_NAME_ENV_VARIABLE_NAME);
+        String bucketName = getEnvironmentVariable(S3_BUCKET_ENV_VARIABLE_NAME);
+        String configFilename = getEnvironmentVariable(CONFIG_FILENAME_ENV_VARIABLE_NAME);
+
+
+        BasicAWSCredentials credentials = new BasicAWSCredentials(accessKey, secretKey);
```

Comment:
Use the default credential provider chain (http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html).  This will then work for environment variables, roles etc.